src/applications/semehr_anon.py

```diff
@@ -44,7 +44,6 @@ def anonymise_dir(input_dir, output_dir, semehr_dir, semehr_anon_cfg_file, write
     cfg_json['anonymisation_output'] = output_dir
     cfg_json['extracted_phi'] = phi_file
     cfg_json['grouped_phi_output'] = '/dev/null'
-    cfg_json['rules_folder'] = os.path.join(semehr_dir, 'CogStack-SemEHR', 'anonymisation/conf/rules')
```

Comment:
If you remove this line then you'll have to remember to edit the config file and hard-code the path to the rules folder.

Reply:
Thanks, and yeah I've done that.